lib/matobj2.gd

```diff
@@ -1926,3 +1926,201 @@ DeclareOperation( "[]:=", [ IsMatrixObj, IsPosInt, IsPosInt, IsObject ] );
 ##    and the available methods have to be adjusted.
 ##
 
+
+############################################################################
+# Elementary matrix operations
+############################################################################
+#
+############################################################################
+##
+##  <#GAPDoc Label="MultMatrixRow">
+##  <ManSection>
+##  <Oper Name="MultMatrixRow" Arg='mat,i,elm'/>
+##  <Oper Name="MultMatrixRowLeft" Arg='mat,i,elm'/>
+##  
+##  <Returns>nothing</Returns>
+##
+##  <Description>
+##  <P/>
+##  Multiplies the <A>i</A>-th row of the mutable matrix <A>mat</A> with the scalar 
+##  <A>elm</A> from the left in-place.
+##  <P/>
+##  <Ref Oper="MultMatrixRowLeft"/> is a synonym of <Ref Oper="MultMatrixRow"/>.
```

Comment:
@wucas @AnnaKDS @danielrademacher any thoughts on this?

Reply:
Sorry, for the late reply @fingolfin.I like it when it's consistent, so I agree. To work on this and the next comment have been on my list since the weekend, but I haven't gotten to it yet. I'll finish this by tomorrow evening.